nuxeo-compound-documents-web/elements/nuxeo-compound-document-tree.js

```diff
@@ -170,6 +170,7 @@ Polymer({
         height: 24px;
         display: table-cell;
         vertical-align: middle;
+        margin-right: 5px;
```

Comment:
I guess this is what you are trying to achieve:
```suggestion
        margin-block-end: 5px;
```
making direction agnostic is helpful for right to left languages.

Reply:
Going with `margin-inline-end: 5px` instead, because the block end is adding the margin to the bottom of the image, not to the right (inline). Thanks for the feedback 🙂